### PR TITLE
[office-js, office-js-preview] Updating PPT APIs with latest from Office-JS CDN

### PR DESCRIPTION
**Pull request by AlexJerabek**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://appsforoffice.microsoft.com/lib/1.1/hosted/office.d.ts https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+     *
+     * Specifies the formatting options for when slides are inserted.
+     *
+     * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    enum InsertSlideFormatting {
+        /**
+         * Copy the source theme into the target presentation and use that theme.
+         *
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91296]: nit: extra line here

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+         */
+        keepSourceFormatting = "KeepSourceFormatting",
+        /**
+         * Use the existing theme in the target presentation.
+         *
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91301]: extra line

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+         * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param index The index of the slide in the collection.
+         * @returns The slide at the given index. An exception is thrown if index is out of range.
+         */
+        getItemAt(index: number): PowerPoint.Slide;
+        /**
+         * Gets a slide using its unique ID. A null object is returned if the slide does not exist.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91438]:
```suggestion
         * Gets a slide using its unique ID. Returns an object whose `isNullObject` property is set to `true` if the slide does not exist.
```

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+         *
+         * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param id The ID of the slide.
+         * @returns The slide with the unique ID. If such a slide does not exist, a null object is returned.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91444]:
```suggestion
         * @returns The slide with the unique ID. If such a slide does not exist, an object whose `isNullObject` property is ste to `true` is returned.
```

File: types/office-js-preview/index.d.ts [hunk 5]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+        interface PresentationLoadOptions {
+            /**
+              Specifying `$all` for the LoadOptions loads all the scalar properties (e.g.: `Range.address`) but not the navigational properties (e.g.: `Range.format.fill.color`).
+             */
+            $all?: boolean;
+            title?: boolean;
+        }
+        /**
+         *
+         * Represents a single slide of a presentation.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91523]: This description doesn't match SlideLoadOptions

File: types/office-js-preview/index.d.ts [hunk 6]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+             * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            id?: string;
+        }
+        /** An interface describing the data returned by calling `slideCollection.toJSON()`. */
+        interface SlideCollectionData {
+            items?: PowerPoint.Interfaces.SlideData[];
+        }
+        /**
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91511]: Description missing for PresentationLoadOptions

File: types/office-js-preview/index.d.ts [hunk 7]
```diff
@@ -91209,9 +91209,386 @@ declare namespace Visio {
...
+             * Gets the unique ID of the slide.
+             *
+             * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            id?: boolean;
+        }
+        /**
+         *
+         * Represents the collection of slides in the presentation.
```
Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:91544]: Description does not match SlideCollectionLoadOptions

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

**Comment by AlexJerabek** [comment]
@Rick-Kirkham, 
I've applied the changes to the *OrNullObject method. The `LoadOptions` feedback is a problem throughout the APIs. Excel and Word have the same description issue. Luckily, these classes are not very visible in the documentation (not included in the TOC). We should determine the proper text and work with @wandyezj to make the changes for the entire codegen process.

I think this PR is ready to check-in.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (AlexJerabek) [reply types/office-js-preview/index.d.ts:91296]: Since this is auto-generated and won't show up in the docs/IntelliSense, I'll leave this in.

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**